src/main/java/org/gridsuite/network/map/dto/utils/ActivePowerControlInfos.java

```diff
@@ -0,0 +1,19 @@
+package org.gridsuite.network.map.dto.utils;
```

Comment:
Its not an util class
Add a package `org.gridsuite.network.map.dto.definition.extension` ?

Reply:
Done